script.py

```diff
@@ -0,0 +1,14 @@
+file = <path to file>
+
+with open(file, "a") as f:
+    for i in range(size_of_your_mom):
+
+        if i%2 == 0:
```

Comment:
please functionally decompose, also lots of new developers dont know about the percent of operator

```suggestion
        if two_percent_of(i) == 0:
```